Library/Formula/pjproject.rb

```diff
@@ -0,0 +1,20 @@
+class Pjproject < Formula
+  homepage "http://www.pjsip.org/"
+  url "http://www.pjsip.org/release/2.3/pjproject-2.3.tar.bz2"
+  sha256 "e7fa60a3b59424430145af90372282ca778449f7b68b77bb24a9cf75d94d5765"
+
+  depends_on 'openssl'
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "dep"
+    system "make"
+    system "make", "install"
+    mkdir bin
+    cp "pjsip-apps/bin/pjsua-x86_64-apple-darwin14.1.0", "#{bin}/pjsua"
```

Comment:
> Where are those documented?

Not entirely sure. They may be technically private methods - The terminology has evolved with Homebrew. I've only been around for a tiny part of that, @jacknagel is probably the guy to talk to about public/private methods, documentation questions, etc. Maybe it is something we should document, maybe it's something we don't formally document for a reason. Jack will know either way.

> # => "10.10"

Indeed. It'll kick out OS X version. ` uname ` is probably the best place to extract the Kernel version itself. But you'll make life easier for yourself if you just ` glob ` for the general name rather than specifics.